classes/Pods.php

```diff
@@ -1015,12 +1015,25 @@ public function field( $name, $single = null, $raw = false ) {
 						$field_names = explode( '.', $params->name );
 
 						if ( isset( $field_names[1] ) ) {
-							$size = $field_names[1];
+							$size    = $field_names[1];
+							$sizes   = get_intermediate_image_sizes();
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 3 spaces
* :no_entry_sign: Intermediate images do not exist on the VIP platform, and thus get_intermediate_image_sizes() returns an empty array() on the platform. This behavior is intentional to prevent WordPress from generating multiple thumbnails when images are uploaded.

<sub>hash: 3a5a3d03c3c6792fd3500e21c65ad453</sub>